cmd/generator/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"os"
+)
+
+func main() {
+	argsWithProg := os.Args
+
+	if len(argsWithProg) != 2 {
+		panic("Wrong amount of arguments!")
+	}
+
+	if len(argsWithProg[1]) == 0 {
+		panic("Given regex is empty!")
+	}
+
+	regex := argsWithProg[1]
+
+	_, err := os.Stdout.WriteString((generateCode(regex) + "\n"))
+	if err != nil {
+		panic("Error writing to stdout")
+	}
+}
+
+func generateCode(regex string) string {
+	afterLastBrace, beforeCurrentBrace, indentations, result := regex, "", "", "rex.New(\n"
+
+	currentOpenBraceIndex, bracesCounter := 0, 0
```

Comment:
```suggestion
	var (
		currentOpenBraceIndex = 0,
		bracesCounter         = 0
	)
```

In order to keep the same style with previous block.

Reply:
Can I join the two blocks like this?

```
var (
		beforeCurrentBrace string
		result             strings.Builder
		afterLastBrace     = regex
		indentations       = ""
		currentOpenBraceI  = 0
		bracesCounter      = 0
	)

```

Reply:
sure, also the default value for `int` is zero, so `bracesCounter int` will work